sds/include/sdsio_client.h

```diff
@@ -36,17 +36,18 @@ typedef struct {
 } header_t;
 
 // Commands
```

Comment:
Change in line 38: "Commands" to "SDSIO Server Command IDs".

You can then remove the additional comments for each command as they do not add value.

Use terms consistently.

"SDSIO Server" is the name of the Server.  Sometimes SDS I/O Server is used.
"ping"  instead of "pings".

Comments can be shorter, for example:

"Initialize SDS I/O Client and Pings SDS I/O Server to verify connection with the server"
-> 
"Initialize SDS I/O Client and ping SDSIO Server to verify connection"